changelog/1.17.3.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "1.17.3"
+description: "Released on 04/09/21"
+---
+
+### Breaking changes ❗
+
+- infra: Removed `tolerations` from workspaces as code templates.
+
+### Features ✨
+
+- web: Added a toggle to enable/disable workspaces as code on the **Manage >
+  Admin > Templates** page. Disabling workspaces as code has the following
+  impact:
+  - New workspaces cannot be created from a template
+  - Existing workspaces built from a template will continue to operate and may
+    be rebuilt. These workspaces will still be notified of changes to remote
+    templates.
+- infra: Added beta support for NVIDIA GPUs in environments created as
+  container-based virtual machines (CVMs).
+- web: Environments can now be created with multiple GPUs (maximum number of
+  GPUs: 20).
```

Comment:
@khorne3 curious if this should be _20_ (referring to https://github.com/cdr/styleguide/blob/master/text-formatting.md#italics), or if this remains without any text emphasis.

Reply:
Oh, I should be more clear in the styleguide -- the italics is for stuff like *x*, so 20 is fine as-is